src/configure.ac

```diff
@@ -3812,6 +3812,17 @@ AC_TRY_COMPILE(
 	AC_MSG_RESULT(yes); AC_DEFINE(HAVE_ST_BLKSIZE),
 	AC_MSG_RESULT(no))
 
+AC_MSG_CHECKING([for clock_gettime])
+AC_TRY_COMPILE([
+#include<time.h>
+#include<sys/time.h>],
+[
+ struct timespec ts;
+ (void)clock_gettime(0, &ts);
+],
+AC_MSG_RESULT(yes); AC_DEFINE(HAVE_CLOCK_GETTIME),
```

Comment:
HAVE_CLOCK_GETTIME also needs to be added to src/config.h.in.
Without it HAVE_CLOCK_GETTIME is never defined.